docs/content/documentation/modules/tensor_mechanics/C0TimoshenkoBeam.md

```diff
@@ -0,0 +1,192 @@
+$$
+
+Note that for the `large_strain` calculation, $A_y$, $A_z$ and $J$ are assumed to be zero for simplicity. This is again valid for symmetric cross-sections such as square, rectangular and circular cross-sections.
+
+## Stress calculation
+The axial and shear strain increments, and the rotational strain increments are passed onto to [ComputeBeamForces](/ComputeBeamForces.md) to calculate the force ($\Delta F$) and moment ($\Delta M$) increments using the linear elastic constitutive relations defined in [ComputeElasticityBeam](/ComputeElasticityBeam.md) as follows:
+$$
+\Delta F_1(x) = E \; \epsilon_{1}(x)
+$$
+$$
+\Delta F_2(x) = kG \; \epsilon_{2}(x)
+$$
+$$
+\Delta F_3(x) = kG \; \epsilon_{3}(x)
+$$
+$$
+\Delta M_1(x) = kG \; \kappa_1(x)
+$$
+$$
+\Delta M_2(x) = E \; \kappa_2(x)
+$$
+$$
+\Delta M_3(x) = G \; \kappa_3(x)
+$$
+where, $E$ and $G$ are the Young's and shear modulus, respectively. $k$ is the shear correction factor that depends on the shape of the cross-section.
+
+Since the strain increments are in the beam local configuration at $t+\Delta t$, the calculated force and moment increments are also in the same configuration. To get the total force and moment at $t + \Delta t$, the force and moment increments are first transformed to the global coordinate system using ${}^{t+\Delta t}R$ and then added to the force and moments from $t$ that are already in the global coordinate system.
+
+The strain energy is calculated as:
+$$
+\delta V = \int_\Omega (\sigma_{11}(x,y,z) \delta \epsilon_{11}(x,y,z) + \sigma_{12}(x,y,z) \delta \epsilon_{12}(x,y,z) + \sigma_{13}(x,y,z) \delta \epsilon_{13}(x,y,z)) \delta \Omega
+$$
+$$
+= \int_{0}^{{}^0L} F_1(x) \delta \epsilon_{1}(x) + F_2(x) \delta \epsilon_{2}(x) + F_3(x) \delta \epsilon_{3}(x) + M_1(x) \delta \kappa_1(x) + M_2(x) \delta \kappa_2(x) + M_3(x) \delta \kappa_3(x)
+$$
+
+As can be seen from the equation for $\epsilon_{1}(x)$, it depends only on $\frac{\partial u_1}{\partial x}$ if $A_y$ and $A_z$ are zero. But $\epsilon_{2}(x)$ depends on both $\frac{\partial u_1}{\partial x}$ and $\theta_3$. Therefore, $F_2(x) \delta \epsilon_{2}(x)$ would contribute to the residual of both the translational dof in y direction and rotational dof in the z direction. Similarly, $F_3(x) \delta \epsilon_{3}(x)$ would contribute to the residual of both the translational dof in z direction and rotational dof in the y direction. This accounts for the coupling between the shear and bending behavior of the beam. The stress-divergence calculation is performed in [StressDivergenceTensorsBeam](/StressDivergenceTensorsBeam.md).
+
+## Dynamic beam
+There are two main ways to include the inertial effects for the beam. The first method is to assign a uniform density to the beam and calculate a consistent mass/inertia matrix for the beam, and the second method assumes the beam to be massless and adds point mass/inertia to the nodes at the end of the beam.
```

Comment:
I think I'd reword the last part of this:
'the second method assumes that the mass of the beam is concentrated at the ends of the beam, and represents the mass of the beam using point mass/inertia at the nodes at the end of the beam.'